src/Test/Utilities/Portable/Syntax/TokenUtilities.cs

```diff
@@ -42,7 +42,7 @@ public static class TokenUtilities
                         }
                     }
 
-                    AssertEx.Fail($"Unexpected token.  Actual '{actualAll}' Expected '{expectedAll}'");
+                    AssertEx.Fail($"Unexpected token.  Actual '{actualAll}' Expected '{expectedAll}'\r\nActual:\r\n{actual}");
```

Comment:
@jasonmalinowski Here's what the output looks like. I don't think it's confusing, but I'm happy to mark the second one as "Actual (full):" if that helps.

```
Test 'Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.CodeRefactorings.ExtractMethod.ExtractMethodTests.TestDeconstruction5' failed:
	Unexpected token.  Actual 'private static void ^NewMethod^ ( out int x , ' Expected 'private static void ^NeMethod^ ( out int x , '
Actual:
class Program
{
    void M()
    {
        int x, y;
        NewMethod(out x, out y);
        System.Console.Write(x + y);
    }

    private static void NewMethod(out int x, out int y)
    {
        (x, y) = (1, 2);
    }
}
Expected: False
Actual:   True
```